code/datums/brain_damage/phobia.dm

```diff
@@ -162,13 +168,19 @@
 			if(fear_state <= PHOBIA_STATE_TERROR)
 				fear_state = PHOBIA_STATE_FAINT
 				owner.remove_movespeed_modifier(MOVESPEED_ID_PHOBIA, TRUE) //in the case that we get so scared by enough bullshit nearby we skip the last stage
-				fearscore = 9
-				owner.visible_message("<span class ='danger'>[owner] faints in fear!</span>", "<span class ='userdanger'>It's too much! You faint!</span>")
-				owner.Sleeping(300)
-				if(prob(stress * 3))
-					owner.set_heartattack(TRUE)
-					to_chat(owner, "<span class='userdanger'>Your heart stops!</span>")
-				stress++
+				if(!timer || COOLDOWN_FINISHED(src, timer))
+					COOLDOWN_START(src, timer, cooldown_length)
+					owner.visible_message("<span class ='danger'>[owner] faints in fear!</span>", "<span class ='userdanger'>It's too much! You faint!</span>")
+					owner.Sleeping(faint_length)
+					fear_state = PHOBIA_STATE_EDGY
+					fearscore = 9
+					stress++
+					if(prob(stress))
+						owner.set_heartattack(TRUE)
+						to_chat(owner, "<span class='userdanger'>Your heart stops!</span>")
+				if(timer && !COOLDOWN_FINISHED(src, timer))
```

Comment:
```suggestion
				else
```

Reply:
Seems like the intention is for these to not run together and for the second one to run if the first one fails

Reply:
Yes, you're right. Didnt think that much into the sequentiality of the code, just the functionality